### PR TITLE
Output CGFRF and CGFRS

### PR DESCRIPTION
**Pull request by totto82**

### CODE REVIEW

**Comment by totto82** [comment]
jenkins build this opm-simulators=5010 please

File: src/opm/output/eclipse/Summary.cpp [hunk 1]
```diff
@@ -2293,6 +2293,8 @@ static const auto funs = std::unordered_map<std::string, ofun> {
     { "CTFAC", trans_factors },
     { "CDFAC", d_factors },
     { "CPI", connection_productivity_index },
+    { "CGFRF", sub(crate<rt::gas, producer>, crate<rt::dissolved_gas, producer>) }, // Free gas flow
+    { "CGFRS", crate<rt::dissolved_gas, producer> },                      // Solution gas flow
```
Comment (bska) [inline src/opm/output/eclipse/Summary.cpp:2297]: Cool.  Maybe you can add
```C++
{ "COFRF", sub(crate<rt::oil, producer>, crate<rt::vaporized_oil, producer> },
{ "COFRS", crate<rt::vaporized_oil, producer> },
```
as well to complete this pair of summary vectors?

**Review by bska — COMMENTED** [review]
Very good.  I think we should add the oil-component split as well, just for the sake of completeness.

**Comment by totto82** [comment]
jenkins build this opm-simulators=5010 please

**Comment by totto82** [comment]
jenkins build this opm-simulators=5010 please

**Comment by totto82** [comment]
jenkins build this opm-simulators=5010 please

**Comment by bska** [comment]
jenkins build this opm-simulators=5010 please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  This looks good to me now and I'll merge into master.

**Pull request merged**